include/Runner.h

```diff
@@ -25,7 +25,9 @@ class Runner {
      * @param input_files   Paths to files to be transformed
      * @param output_path   Path to the output dir where transformed code will be stored
      */
-    static void createOutputFolders(int num_files, char * input_files[], const string& output_path);
+    map<int, char **> createOutputFolders(int num_files, char * input_files[], const string& output_path);
```

Comment:
How many elements would be stored in this map? If not many, I think a simple vector would be faster. Otherwise, maybe unordered_map is the best choice.

Reply:
Also, maybe it's better to pass the container as an argument. Like it was done in regex functions.